### DISCUSSION TITLE
Responsive default template

### DISCUSSION THREAD

**Discussion by markusgritsch** [lona-web-org/lona#377] [Ideas]
What do you think about adding `<meta name="viewport" content="width=device-width, initial-scale=1">` to the default Lona `frontend.html`? (It's already present in the `base.html` from `lona-picocss`.)

It is recommended e.g. [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Viewport_meta_tag) and [here](https://www.w3schools.com/css/css_rwd_viewport.asp) to get pages properly rendered on mobile devices.

**Comment by fscherf** [comment]
Sounds reasonable. Do you want to open a PR for this?

**Reply by markusgritsch** [reply]
-> PR: https://github.com/lona-web-org/lona/pull/378
